modules/virt-about-vm-snapshots.adoc

```diff
@@ -18,6 +18,7 @@ The snapshot stores a copy of each Container Storage Interface (CSI) volume atta
 You can perform the following snapshot actions:
 
 * Create a new snapshot
+* Clone a snapshot to create a VM
```

Comment:
There's no such action as 'Clone a snapshot'
VirtualMachineSnapshot can be created to snapshot the VM.
VirtualMachineRestore can be created to restore the VM (to the same or to a different name).
VirtualMachineSnapshot can't be Cloned.

VM can be cloned with VirtualMachineClone

I opened a bug to the UI: https://issues.redhat.com/browse/CNV-38331

Reply:
I've updated the bullet to "Create a copy of a virtual machine from a snapshot."